web-src/src/components/ModalDialogPlaylistSave.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <transition name="fade">
+      <div class="modal is-active" v-if="show">
+        <div class="modal-background" @click="$emit('close')"></div>
+        <div class="modal-card">
+          <section class="modal-card-body">
+            <form v-on:submit.prevent="save">
+              <div class="field">
+                <p class="control is-expanded has-icons-left">
+                  <input class="input is-rounded is-shadowless" type="text" placeholder="playlist name" v-model="pls_name" ref="pls_name_field">
+                  <span class="icon is-left">
+                    <i class="mdi mdi-file-music"></i>
+                  </span>
+                </p>
+              </div>
+            </form>
+          </section>
+          <footer class="modal-card-foot">
+            <button class="button is-success" @click="save" :disabled="pls_name.length < 3">Save</button>
```

Comment:
Why is it required to have a name of at least 3 characters?

Reply:
originally i was thinking user needed to incl the named extension (ie .pls) before the move to using the `library_save()`.  reduced this down to a min of 1 char for playlist name